samples/zh-tw/Getting Started/main.css

```diff
@@ -32,7 +31,7 @@ a {
 }
 
 b, strong {
-    font-weight: 700;
+    font-weight: 600;
```

Comment:
`font-weight: 700` is now `font-weight: 600`. It matches the others, but might be worth it to double check if the `zh-tw` font requires  the heavier weight.